### ISSUE TITLE
Error in Migration/RemoveColumn when table name is variable

### ISSUE THREAD

**Issue by Darhazer** [r7kamura/rubocop-migration#11]
Having a migration like:
```ruby
TABLES = %I[
  foo
  bar
].freeze

def change
    TABLES.each do |table_name|
      remove_column table_name, :deleted_at, :datetime
    end
end
```

Migration/RemoveColumn errors with:
```
undefined method `value' for s(:lvar, :table_name):RuboCop::AST::Node
rubocop-migration-0.4.1/lib/rubocop/cop/migration/remove_column.rb:138:in `find_table_name_from'
rubocop-migration-0.4.1/lib/rubocop/cop/migration/remove_column.rb:145:in `ignored?'
rubocop-migration-0.4.1/lib/rubocop/cop/migration/remove_column.rb:107:in `bad?'
rubocop-migration-0.4.1/lib/rubocop/cop/migration/remove_column.rb:47:in `on_send'
```

**Comment by r7kamura** [comment]
Thanks for reporting back. It appears that the following code need to be fixed:

https://github.com/r7kamura/rubocop-migration/blob/5d7f9b5a50cf3df6e60409d0015f1a599c861158/lib/rubocop/cop/migration/remove_column.rb#L135-L139

It seems that current implementation believes that `node.arguments[0]` is a literal node, so you are getting an error when it is not.

**Comment by r7kamura** [comment]
Fixed at https://github.com/r7kamura/rubocop-migration/pull/12 and just released v0.4.2 👌 

- https://github.com/r7kamura/rubocop-migration/releases/tag/v0.4.2
- https://rubygems.org/gems/rubocop-migration/versions/0.4.2

**Comment by Darhazer** [comment]
Thank you 🙇
